nginx.conf

```diff
@@ -54,6 +54,16 @@ http {
         listen 80;
         root /fm;
 
+        location /api/ {
+            proxy_set_header X-Real-IP $remote_addr;
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_set_header X-NginX-Proxy true;
+            proxy_pass http://localhost:1337/api/;
```

Comment:
nope, there will be another proxy infront of this that will direct `/api` traffic the Frontend API.